edb/schema/pointers.py

```diff
@@ -1717,10 +1719,19 @@ def _cmd_tree_from_ast(
         context: sd.CommandContext,
     ) -> sd.Command:
         cmd = super()._cmd_tree_from_ast(schema, astnode, context)
+        assert isinstance(cmd, PointerCommand)
+
         referrer_ctx = cls.get_referrer_context(context)
         if referrer_ctx is not None:
             if getattr(astnode, 'declared_overloaded', False):
                 cmd.set_attribute_value('declared_overloaded', True)
+        else:
+            # This is an abstract propery/link
```

Comment:
```suggestion
            # This is an abstract property/link
```

Reply:
hey I finally figured out how to make a suggestion!

Reply:
My spell checker only shows first 100 mistakes in a file :confused: